docs/JavaCodePerformance.md

```diff
@@ -2521,6 +2521,73 @@ class SingletonBMGood {
 }
 ```
 
+#### TUTC14
+**Observation: Locking and describing the locking behaviour with `@GuardedBy` is applied in a wrong way or not meeting best practices.**   
+**Problem:** Wrong use of locking and `@GuardedBy` may result in thread-unsafety and concurrency bugs. It may obstruct other GuardedBy rules which check whether the promise of guarding is actually met, it may disguise high risk violations.  
+**Solution:** Use `@GuardedBy` properly: 
+* use a private final lock object constructed with `new Object()` or `new Object[0]` 
```

Comment:
what about using `java.util.concurrent.locks.*Lock` variants? Use of these will cause false positives?

Reply:
Those are not supported, yet. Don't know what will happen.

Reply:
OK, let's put this in scope of another issue/fix request when the need arrises.